### PR TITLE
Extension of documentation

### PR DESCRIPTION
**Pull request by mseeger**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/launch_huggingface_classification.py [hunk 1]
```diff
@@ -77,7 +77,9 @@
         config_space, mode=mode, metric=metric, random_seed=random_seed
     )
 
-    stop_criterion = StoppingCriterion(max_wallclock_time=3000) # wall clock time can be increased to 1 hour for more performance
+    stop_criterion = StoppingCriterion(
```
Comment (mseeger) [inline examples/launch_huggingface_classification.py:80]: The previous PR must not have run black properly

File: docs/source/tutorials/developer/random_search.rst [hunk 2]
```diff
@@ -101,15 +101,25 @@ should be used by all searchers which make random decisions. It maintains a PRN
...
-from this class, or avoid duplicate suggestions in a different way. While we
-in general recommend to use the default ``allow_duplicates == False``,
-allowing for duplicates can be useful when dealing with configuration spaces of
-small finite size.
+extends ``SearcherWithRandomSeed``. It supports a number of features which are
+desirable for most searchers:
 
-Finally, let us walk through
+* Seed management for random decisions.
+* Avoid to suggest the same configuration more than once. While we in general
```
Comment (wesk) [inline docs/source/tutorials/developer/random_search.rst:108]:
```suggestion
* Avoid suggesting the same configuration more than once. While we in general
```

File: docs/source/tutorials/developer/wrap_code.rst [hunk 3]
```diff
@@ -118,6 +129,12 @@ encoded as vectors with values in :math:`[0, 1]`, which is done using the
    of finite size, and uses the random generator seeded in a consistent and
    reproducible way.
 
+   We also recommend to implement the ``restrict_configurations`` argument,
```
Comment (wesk) [inline docs/source/tutorials/developer/wrap_code.rst:132]:
```suggestion
   We also recommend that you implement the ``restrict_configurations`` argument,
```

File: examples/launch_huggingface_classification.py (hunk 1 above)
Comment (wesk) [reply examples/launch_huggingface_classification.py:80]: That was me - seems like it's not captured in our package linter rule, I will take a look next week

**Review by wesk — COMMENTED** [review]
Minor phrasing suggestions

File: examples/launch_huggingface_classification.py (hunk 1 above)
Comment (wesk) [reply examples/launch_huggingface_classification.py:80]: Thanks for flagging this, my miss

**Review by wesk — APPROVED** [review]

**Pull request merged**